frontend/src/pages/Graph/GraphToolbar/GraphFind.tsx

```diff
@@ -162,6 +163,12 @@ export class GraphFind extends React.Component<GraphFindProps, GraphFindState> {
       }
     } else if (!!findValue) {
       HistoryManager.setParam(URLParam.GRAPH_FIND, findValue);
+    } else {
+      serverConfig.kialiFeatureFlags.uiDefaults.graph.findOptions.forEach(opt => {
+        if (opt.autoEnable === true) {
+          props.setFindValue(opt.expression);
+        }
+      });
```

Comment:
```suggestion
      const autoSelect = serverConfig.kialiFeatureFlags.uiDefaults.graph.findOptions.find(opt => opt.autoSelect);
      if (autoSelect) {
        props.setFindValue(opt.expression);
      }
```

This is minor, but in general I suggest not using Array.ForEach when you don't really want to perform an action on each array element.  In this case you only want to perform autoSelect one time, even if it is defined more than once.